### PR TITLE
Fix the blurhash alignment and general style

### LINKED ISSUE #1397: Misaligned images
- The left image has smaller height
- The toggle icon seems to placed lower than it should

Can be reproduced with https://mastodon.me.uk/@mdales/102636326167158035

image

### PR DESCRIPTION
**Pull request by sgenoud**
I tried to align it with the style without blurhash (rounded corders, alignment).

I also moved the button on the left (it seemed a bit weird with two different spacing).

Capture d’écran 2019-08-21 à 17 42 40

Closes #1397

### CODE REVIEW

File: src/routes/_components/status/StatusMediaAttachments.html [hunk 1]
```diff
@@ -233,7 +236,21 @@
       sensitiveMediaIconClass: ({ canUseBlurhash }) => classname(
         'status-sensitive-media-svg',
         canUseBlurhash && 'status-sensitive-media-svg-transparent'
-      )
+      ),
+      nCols:
+        ({ mediaAttachments, $largeInlineMedia }) => {
+          return (!$largeInlineMedia && mediaAttachments.length > 1) ? 2 : 1
+        },
```
Comment (nolanlawson) [inline src/routes/_components/status/StatusMediaAttachments.html]: Is this `nCols` unused?

**Comment by nolanlawson** [comment]
Tested locally, looks good to me! Thanks much for this. :)

File: src/routes/_components/status/StatusMediaAttachments.html (hunk 1 above)
Comment (nolanlawson) [reply src/routes/_components/status/StatusMediaAttachments.html]: It's OK, I can remove it in another PR :)

Comment (sgenoud) [reply src/routes/_components/status/StatusMediaAttachments.html]: Yes we can remove it - I must admit I did just copy the code and asumed that twoCols depended on nCols...

**Pull request merged**